wcfsetup/install/files/lib/system/worker/UserContentRemoveWorker.class.php

```diff
@@ -49,18 +51,34 @@ class UserContentRemoveWorker extends AbstractWorker {
 	 * @inheritDoc
 	 */
 	public function validate() {
-		if (!isset($this->parameters['userID'])) {
-			throw new \InvalidArgumentException('userID missing');
+		if (isset($this->parameters['userID']) && !isset($this->parameters['userIDs'])) {
+			$this->parameters['userIDs'] = [$this->parameters['userID']];
 		}
 		
-		$this->user = new User($this->parameters['userID']);
-		
-		if (!$this->user->userID) {
-			throw new \InvalidArgumentException('userID is unknown.');
+		if (isset($this->parameters['userIDs']) && is_array($this->parameters['userIDs']) && !empty($this->parameters['userIDs'])) {
+			$userList = new UserList();
+			$userList->setObjectIDs($this->parameters['userIDs']);
+			$userList->readObjects();
+			
+			if ($userList->count() !== count($this->parameters['userIDs'])) {
+				$diff = array_diff($this->parameters['userIDs'], array_map(function (User $user) {
+					return $user->userID;
+				}, $userList->getObjects()));
```

Comment:
```suggestion
				$diff = array_diff($this->parameters['userIDs'], array_column($userList->getObjects(), 'userID'));
```

Could apply likewise to `$knownContentProvider` below.